pkg/graph/schema.resolvers.go

```diff
@@ -1949,6 +1998,24 @@ func (r *systemIntakeDocumentResolver) CanView(ctx context.Context, obj *models.
 	return resolvers.CanViewDocument(ctx, grbUsers, obj), nil
 }
 
+// VotingRole is the resolver for the votingRole field.
+func (r *systemIntakeGRBReviewDiscussionPostResolver) VotingRole(ctx context.Context, obj *models.SystemIntakeGRBReviewDiscussionPost) (*models.SystemIntakeGRBReviewerVotingRole, error) {
+	if obj.VotingRole == nil {
+		return nil, nil
+	}
+	strVal := *obj.VotingRole
+	return helpers.PointerTo(models.SystemIntakeGRBReviewerVotingRole(strVal)), nil
+}
+
+// GrbRole is the resolver for the grbRole field.
+func (r *systemIntakeGRBReviewDiscussionPostResolver) GrbRole(ctx context.Context, obj *models.SystemIntakeGRBReviewDiscussionPost) (*models.SystemIntakeGRBReviewerRole, error) {
+	if obj.GRBRole == nil {
+		return nil, nil
+	}
+	strVal := *obj.GRBRole
+	return helpers.PointerTo(models.SystemIntakeGRBReviewerRole(strVal)), nil
+}
```

Comment:
Any idea why we have/need two distinct types for `System Intake GRB Reviewer Role` and `System Intake GRB Reviewer Voting Role`?

Seems like we have a copy of each, one that's auto-generated, another that's hand-written, and this resolver's just converting between the two.

Reply:
Just tested locally -- looks like modifying these lines as noted causes those resolvers to just drop off!
https://github.com/CMS-Enterprise/easi-app/blob/da34950adc5a4e7a73aadfe02f01642e65b151c0/pkg/models/system_intake_grb_discussions.go#L7-L14

```go
type SystemIntakeGRBReviewDiscussionPost struct {
	BaseStructUser
	Content        HTML                               `json:"content" db:"content"`
	SystemIntakeID uuid.UUID                          `json:"systemIntakeId" db:"system_intake_id"`
	ReplyToID      *uuid.UUID                         `db:"reply_to_id"`
	VotingRole     *SystemIntakeGRBReviewerVotingRole `json:"votingRole" db:"voting_role"`
	GRBRole        *SystemIntakeGRBReviewerRole       `json:"grbRole" db:"grb_role"`
}
```

Reply:
Per a 1:1 with Lee, we're gonna save this work for another PR since this crops up in a few places